citadel/indico_citadel/util.py

```diff
@@ -104,7 +104,7 @@ def format_filters(params, filters, range_filters):
     :param params: The filter query params
     :param filters: The filter whitelist
     :param range_filters: The range filter whitelist
-    :returns: filters, extracted placeholders
+    :returns filters, extracted placeholders
```

Comment:
I don't think this is correct; sphinx uses `:returns: whatever` and `:return: whatever` (AFAIK we use the second version more frequently)

Reply:
Thanks, we had different formats and I had the impression you would point it out if I didn't choose the right one 🙂